client/src/App.js

```diff
@@ -1,45 +1,39 @@
-import React from 'react';
+import React, {Component} from 'react';
 import SelectedFoods from './SelectedFoods';
 import FoodSearch from './FoodSearch';
 
-const App = React.createClass({
-  getInitialState: function () {
-    return {
-      selectedFoods: [],
-    };
-  },
-  render: function () {
+class App extends Component {
+  state = {
```

Comment:
Isn't this a stage 2 feature that isn't actually "ES6" (as you mention in the PR title)? 
* https://babeljs.io/docs/plugins/transform-class-properties/
* https://github.com/tc39/proposal-class-public-fields

If we don't want to use this syntax, we could use the actual ES6 way of defining a `constructor` and setting `this.state` in there:

```js
constructor(props) {
  super(props);
  this.state = {
    selectedFoods: []
  }
}
```